knimin/tests/test_ag_edit_barcode.py

```diff
@@ -8,6 +8,10 @@
 
 
 class TestAGEditBarcodeHandler(TestHandlerBase):
+    # these fields switch None representation and thus are hard to check
+    # for equality
+    none_fields = ['environment_sampled', 'withdrawn', 'refunded']
```

Comment:
What do you mean that they change None representation? If they go from some value to None and that's the expected behavior you can check that they're actually `None`, otherwise they go untested.